src/CoreMidi/MidiStructs.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Runtime.InteropServices;
+
+using ObjCRuntime;
+using CoreFoundation;
+using Foundation;
+
+
+namespace CoreMidi {
+
+#if NET
+		[SupportedOSPlatform ("ios14.0")]
+		[SupportedOSPlatform ("maccatalyst")]
```

Comment:
Version?

Reply:
No need, macCatalyst min version is 14.0